### PR TITLE
added docs for activiityLog

### PR DESCRIPTION
**Pull request by soquasi**

### CODE REVIEW

File: docs/api/devguide/activitylogguide.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+## Written features
+***
+The features written to the new space are stored by uuid. So the space with this feature enabled needs to have **enableUUID** set to true.
+
+Additionally, the original feature element will be slightly modified:
+
+* The id of the feature, will be the **uuid** of the modified feature in the original space.
+* The original properties of **'@ns:com:here:xyz'** will be stored under '@ns:com:here:xyz'.**original**
+* The original id of the feature will be stored under '@ns:com:here:xyz'.**original**.**id** within the properties.
+* A new property will be added under '@ns:com:here:xyz'.**original**.**invalidAt** to indicate the time range, when this object was the HEAD (newest) history object.
```
Comment (cfjedimaster) [inline docs/api/devguide/activitylogguide.md]: Query - why "invalidAt"? The name doesn't seem to match the intent. Also, what kind of data is this? You say "time range", is it in an array of timestamps?

File: docs/api/quickstart/activitylog.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+          "geometry":{
+            "type":"Point",
+            "coordinates":[1,0]
+          }
+        }
+      ]
+    }
+    ```
+
+    Check into your activity log space:
```
Comment (cfjedimaster) [inline docs/api/quickstart/activitylog.md]: remove "into"

File: docs/api/quickstart/activitylog.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+          }
+        }
+      ]
+    }
+    ```
+
+## Search for specific feature
+
+You can search the Activity Log for a specific feature using its original id.
+This request returns an unsorted list of all revisions of the object.
```
Comment (cfjedimaster) [inline docs/api/quickstart/activitylog.md]: Interesting - why isn't it sorted by the time the change happened?

File: docs/api/quickstart/activitylog.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+      }
+    }
+  ]
+}
+```
+
+## Look up a specific revision of a feature
+
+You can search for a specific revision of a feature using the *uuid* of the feature:
+
```
Comment (cfjedimaster) [inline docs/api/quickstart/activitylog.md]: Yeah this confuses me. Does this return the most recent revision? Doesn't every revision have the same UUID from the original feature? What exactly is being searched here?

File: docs/api/devguide/activitylogguide.md (hunk 1 above)
Comment (soquasi) [reply docs/api/devguide/activitylogguide.md]: It's not the latest version of the documentation. E.g., the latest wording is invalidatedAt, not invalidAt.

**Pull request merged**